cmd/kube-proxy/app/conntrack.go

```diff
@@ -112,7 +112,7 @@ func isSysFSWritable() (bool, error) {
 	m := mount.New("" /* default mount path */)
 	mountPoints, err := m.List()
 	if err != nil {
-		klog.Errorf("failed to list mount points: %v", err)
+		klog.ErrorS(err, "failed to list mount points")
```

Comment:
```suggestion
		klog.ErrorS(err, "Failed to list mount points")
```

Reply:
the kep seems to prefer caps too https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging#new-klog-methods